elasticai/creator/vhdl/language.py

```diff
@@ -99,6 +103,92 @@ def __init__(self, identifier: str):
         super().__init__(identifier, Keywords.COMPONENT)
 
 
+class Architecture:
+    def __init__(self, identifier: str, design_unit: str, process_content: str):
+        self.identifier = identifier
+        self.design_unit = design_unit
+        self.process_content = process_content
+
+    def _header(self) -> Code:
+        yield self.process_content
+
+    def __call__(self) -> Code:
+        return _wrap_in_ARCHITECTURE_block(
```

Comment:
this seems weird... if it's an architecture block, why do we pass the architecture keyword to it?